dbt/include/trino/macros/adapters.sql

```diff
@@ -151,6 +151,11 @@
 {% endmacro %}
 
 
+{% macro trino__get_batch_size() %}
```

Comment:
I recommend adding a documentation for this macro and its purpose in `dbt seed` command.

Please also point out in the documentation that the user can "override" this default behaviour by adding `default__get_batch_size()` macro definition on his project.
Or if you think that this is way too verbose in the code, you could add something similar to the `README.md` file in a separate section.

Reply:
Done